app/screens/Appointment/AppointmentSchedule.js

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Button from '../../components/Button/Button';
+import DateTimePicker from '../../components/DateTimePicker/DateTimePicker';
+import Dialog from '../../components/Dialog/Dialog';
+import TextField from '../../components/FormField/TextField';
+import Table from '../../components/Table/Table';
+
+import BaseScreen from '../BaseScreen';
+
+const StyledSearchTextField = styled(TextField)`
+  &&& {
+    margin: 1em 0;
+  }
+`;
+
+export default class AppointmentSchedule extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      appointment: {
+        schedule: new Date()
```

Comment:
`new Date()` introduces a side-effect, is there a way to eliminate this?

Reply:
I don't have any idea right now

Reply:
I think this is fine for now